### PR TITLE
Add Modal to Deposit USDC to Aave and get stataUSDC.

### PR DESCRIPTION
**Pull request by lakshya-sky**
Adds a Modal to convert USDC to stataUSDC for Base Andromeda

### CODE REVIEW

File: contracts/importers/importStataUSDC.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+export async function importStataUSDC(chainId, preset) {
+  if (!preset) {
+    throw new Error(`Missing preset`);
+  }
+  const deployment = `${Number(chainId).toFixed(0)}-${preset}`;
+  switch (deployment) {
+    case '8453-andromeda': {
+      const [{ default: meta }, { default: abi }] = await Promise.all([
+        import('@synthetixio/v3-contracts/8453-andromeda/meta.json'),
+        import('./stataUSDC.json'),
```
Comment (lakshya-sky) [inline contracts/importers/importStataUSDC.js:10]: Right now I used a local file for the abi but once this PR is approved we need to update and use https://github.com/Synthetixio/v3-contracts/blob/master/8453-andromeda/CollateralToken_stataBasUSDC.readable.json

Comment (noisekit) [reply contracts/importers/importStataUSDC.js:10]: That was merged 2 mo ago and already available in liquidity ui

Comment (lakshya-sky) [reply contracts/importers/importStataUSDC.js:10]: that one doesn't have deposit method in the abi

Comment (noisekit) [reply contracts/importers/importStataUSDC.js:10]: Yeh those ones have standard erc20 abi.

**Comment by noisekit** [comment]
I believe this is moved to #9

**Pull request closed without merging**